src/TLuaInterpreter.cpp

```diff
@@ -7112,7 +7111,7 @@ void TLuaInterpreter::createCookiesTable(lua_State* L, QNetworkReply* reply)
 
     // Parse cookies, add them as key-value pairs to the empty table
     const Host& host = getHostFromLua(L);
-    QNetworkCookieJar* cookieJar = host.mLuaInterpreter.mpFileDownloader->cookieJar();
+    QNetworkCookieJar* cookieJar = host.getNetworkAccessManager()->cookieJar();
```

Comment:
Since we are only looking into the jar and not  adding or stealing them:
```suggestion
    const QNetworkCookieJar* cookieJar = host.getNetworkAccessManager()->cookieJar();
```